src/renderer.zig

```diff
@@ -917,8 +917,14 @@ pub const MeshSelection = struct { // MARK: MeshSelection
 		while(total_tMax < closestDistance) {
 			const block = mesh_storage.getBlock(voxelPos[0], voxelPos[1], voxelPos[2]) orelse break;
 			if(block.typ != 0) blk: {
+				var fluidPlaceable = false;
+				if(item) |realItem| {
+					if(realItem == .baseItem and realItem.baseItem.hasTag(.fluidPlaceable)) {
+						fluidPlaceable = true;
+					}
+				}
```

Comment:
At this point we could just turn it into a direct assignment:
```suggestion
				const fluidPlaceable = item != null and item.? == .baseItem and item.?.baseItem.hasTag(.fluidPlaceable);
```

This will likely turn directly into `item.hasTag` after #1443 anyways.